server/src/main/java/org/apache/cloudstack/direct/download/DirectDownloadManagerImpl.java

```diff
@@ -19,6 +19,10 @@
 package org.apache.cloudstack.direct.download;
 
 import static com.cloud.storage.Storage.ImageFormat;
+import static org.apache.cloudstack.direct.download.DirectDownloadManager.HostCertificateStatus.CertificateStatus.FAILED;
+import static org.apache.cloudstack.direct.download.DirectDownloadManager.HostCertificateStatus.CertificateStatus.REVOKED;
+import static org.apache.cloudstack.direct.download.DirectDownloadManager.HostCertificateStatus.CertificateStatus.SKIPPED;
+import static org.apache.cloudstack.direct.download.DirectDownloadManager.HostCertificateStatus.CertificateStatus.UPLOADED;
```

Comment:
possible to import _org.apache.cloudstack.direct.download.DirectDownloadManager.HostCertificateStatus.CertificateStatus_ , and use the status enums ?